src/constants/common.js

```diff
@@ -107,7 +107,34 @@ module.exports = {
 	PUBLIC_ROLE: 'public',
 	RESOURCE_TITLE: 'title',
 	BODY: 'body',
+	TASK_EVIDENCE: 'evidence_details',
 	REQUEST_METHOD_DELETE: 'DELETE',
 	FILE_TYPE: 'file_types',
 	CHILDREN: 'children',
+	TASK_ALLOWED_FILE_TYPES: 'allowed_file_types',
+	ALLOWED_FILE_TYPES: {
+		Images: ['jpg', 'png', 'jpeg', 'BMP', 'GIF', 'TIFF', 'HEIF'],
```

Comment:
@adithyadinesh0412 make it to lowercase and ask the frontend to send that from entities